src/web/app/components/question-text-with-info/question-text-with-info.component.spec.ts

```diff
@@ -30,4 +32,37 @@ describe('QuestionTextWithInfoComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should not show control link and question details for TEXT questions', () => {
```

Comment:
But you did not set the question type to TEXT before the test? You cannot rely on the default value.

Reply:
You cannot just set the questionType to TEXT. It is possible that the internal structure is not of question type text. It is better to set the whole questionDetails.